src/Controller/AdminController.php

```diff
@@ -1490,6 +1492,98 @@ public function apiAppAction($id, Request $request)
         ]);
     }
 
+    /**
+     * @Route("/admin/integrations", name="admin_integrations")
```

Comment:
Should be `/admin/integrations/woopit` + `admin_integrations_woopit`. 
`/admin/integrations` should exist, but just show "Woopit".